workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css

```diff
@@ -888,6 +892,12 @@ pre[class*='language-'] {
 	}
 }
 
+.learn-category div {
+	color: #282934;
```

Comment:
spacing, and use variable colors from/liferay-portal/workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/style-books/liferay-learn/frontend-tokens-values.json